code/game/gamemodes/cult/cult.dm

```diff
@@ -94,7 +95,8 @@ GLOBAL_LIST_EMPTY(all_cults)
 	..()
 	var/list/cultists_possible = get_players_for_role(ROLE_CULTIST)
 
-	for(var/cultists_number = 1 to max_cultists_to_start)
+	cultist_amount = min(round(num_players() / (required_players / min_cultists_to_start)), max_cultists_to_start)
+	for(var/cultists_number = 1 to cultist_amount)
```

Comment:
While this apparently works, it should probably be changed to the more common syntax of:
```js
for(var/cultists_number in 1 to cultist_amount)
```
Using `in` over `=`.